package.json

```diff
@@ -85,6 +85,7 @@
 		"express-session": "^1.17.1",
 		"glob": "^7.1.4",
 		"helmet": "^4.2.0",
+		"immer": "^9.0.12",
 		"immutable": "^4.0.0-rc.12",
```

Comment:
Lets remove immutable since its no longer being used

Reply:
Oh right. I told myself I would and totally forgot. Good catch